src/components/Meme.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react"
+
+export default function Meme() {
+
+    // const [memeImage, setMemeImage] = React.useState("")
+    const [meme, setMeme] = useState({
+        topText : "",
+        bottomText : "",
+        randomImg : "http://i.imgflip.com/1bij.jpg"
+    })
+    const [allMemes, setAllMemes] = useState([])
+
+    useEffect(()=>{
+        fetch("https://api.imgflip.com/get_memes")
+        .then(res => res.json())
+        .then(data => setAllMemes(data.data.memes))
+    }, [])
+
+    function getMemeImage() {
+        const randomNumber = Math.floor(Math.random() * allMemes.length)
+        const url = allMemes[randomNumber].url
```

Comment:
Try to use destructuring  to avoid the repetition  it will make the code much clear and readable.
```suggestion
        const {url} = allMemes[randomNumber]
```